kagami/database/database_service/announce.py

```diff
@@ -42,3 +43,6 @@ async def update_announce(
         if new_title:
             origin_announce.title = new_title
         await self._session.commit()
+
+    async def list_announce(self) -> list[Announce]:
+        return await self._session.execute(select(Announce))
```

Comment:
Is there any limit on the amount of quried data? Consider add a paging to this.

Reply:
That will have some changes on api definition. cc @GEORGE-Ta

Reply:
I am thinking about adding page number and page size in api definition.

Reply:
[2e95610](https://github.com/ProjectSoftoid/Kagami/pull/26/commits/2e9561006aa2298864a7bcee87d4fabc6e8b43bf)